### ISSUE TITLE
For Reference Only: HIF Emergency Housing Fund 2015 Forms

### ISSUE THREAD

**Issue by lamplus** [on-site/grantzilla#240]
These documents are here for reference only.  This ticket does not represent any work.

[hif_emergency_housing_fund_2015.pdf](https://github.com/on-site/grantzilla/files/175261/hif_emergency_housing_fund_2015.pdf)
[hif_online_application.pdf](https://github.com/on-site/grantzilla/files/175263/hif_online_application.pdf)

**Comment by scottjones** [comment]
I'm working on adding these forms.

**Comment by lamplus** [comment]
Scott,
Kevin and I have been working on the Source of Income and Property pages.  The changes are in a temporary branch (temp-source-of-income-page) with both or issue-216-property-information-page and issue-237-sources-of-income-page.  We started but can't get the form to save the data to the db and or displayed on the page.

**Comment by scottjones** [comment]
Do they actually want us to be able to print these PDFs?

**Comment by lamplus** [comment]
Yes, they need the print feature (I thought we have an issue created for printing feature).  The last page, Client Certification, requires signature.  We are thinking to have printed pages for now and eventually may be implementing the esign (if possible).

**Comment by scottjones** [comment]
So, is this ticket here just to have a place for the full application forms?

**Comment by scottjones** [comment]
I want to make sure I'm working on the thing you've prioritized as important!  :smile:

**Comment by lamplus** [comment]
Yes, this issue is a placeholder for storing the documentations/PDFs that we received for references.  Sorry that I didn't state it in the issue.

There should be issues created for each individual page (if not, will be created soon).

**Comment by scottjones** [comment]
OK, I'll unassign and work on helping with your existing branches mentioned above.

**Comment by lamplus** [comment]
Thanks Scott.
